Doc/library/base64.rst

```diff
@@ -135,6 +135,13 @@ The modern interface provides:
    incorrectly padded or if there are non-alphabet characters present in the
    input.
 
+.. function:: b32hexencode(s)
+
+   Similar to :func:`b32encode` but uses the Extended Hex Alphabet.
```

Comment:
Maybe a link to the RFC could be added? It's not clear what the Extended Hex Alphabet is and which alphabet you should be using when you open the documentation.

Reply:
Also, I think we can save it for another PR, but the `base64` documentation is pretty sparse, it might be nice to bring in some of the information available in the RFCs, like the tables of the alphabets and whatnot.

Reply:
The RFC 3548 reference here is only used for `map01`. RFC 4648 does not define this interchangable characters, probably because `0` and `O` or `1` and `I` are different in the hex alphabet. This RFC 3548 should stay but will only show up in the `b32decode` docstring to describe the `map01` argument.

Reply:
RFC 4648 obsoletes RFC 3548, so you can replace even the one reference to `map01` to the new RFC. It's in [section 3.4](https://tools.ietf.org/html/rfc4648#section-3.4), and yeah it's only supported in the `base32` alphabet (which deliberately leaves out 0 and 1 to avoid this confusion).

There are multiple references to RFC 3548 that should now be cleaned up in the documentation — having actual support for RFC 4648 (which this adds) is what blocked PR #2336.

Reply:
Ah, sorry. Missed that in the RFC.